README.md

```diff
@@ -144,20 +144,13 @@ spec:
 ```
 
 # Using
-The operator looks for a CRD of type `NetworkConfig` with the name `default`. Whenever this object is created or changed, the operator will apply those changes to the cluster (if they are valid and safe to do so).
-
-You can view and edit that object with
-```
-oc edit networkconfig cluster
-```
-
-and any changes will be automatically applied.
+The operator is expected to run as a pod (via a Deployment) inside a kubernetes cluster. It will retrieve the configuration above and reconcile the desired configuration. A suitable manifest for running the operator is located in `manifests/`.
 
 ## Unsafe changes
 Most network changes are unsafe to roll out to a production cluster. Therefore, the network operator will stop reconciling if it detects that an unsafe change has been requested. 
 
 ### Safe changes to apply:
-It is safe to edit the following fields in `NetworkConfig.Spec`:
+It is safe to edit the following fields in the Operator configuration:
```

Comment:
In the safe and unsafe configurations, I think you make changes by editing the cluster and operator config objects and save them. Is that so?  If yes, how do you back out errors in an incorrect config? How does the problem show in the status?